suite-native/receive/src/components/DeviceScreen/DeviceScreenContent.tsx

```diff
@@ -35,21 +37,21 @@ type DeviceModelLayoutProps = {
 
 const deviceToContentStyles = {
     [DeviceModelInternal.T1B1]: {
-        fontSource: require('../../../../packages/theme/fonts/PixelOperatorMono8-Regular.ttf'),
+        fontSource: require('../../../../../packages/theme/fonts/PixelOperatorMono8-Regular.ttf'),
```

Comment:
you can use just `@trezor/theme/fonts/....` so you will avoid relative path

Reply:
9fe8f1e341